doc/man1/openssl-verify.pod.in

```diff
@@ -326,63 +123,6 @@ Certificates must be in PEM format.
 
 =back
 
-=head1 VERIFY OPERATION
-
-This command uses the same functions as the internal SSL
-and S/MIME verification, therefore this description applies to these verify
-operations too.
-
-There is one crucial difference between the verify operations performed
-by this command: wherever possible an attempt is made to
-continue after an error whereas normally the verify operation would halt on
-the first error. This allows all the problems with a certificate chain to be
-determined.
```

Comment:
Is the above paragraph about the verification mode in verify(1) preserved somewhere?

Reply:
Yes, as described in the openssl command, okay?